packages/menu/src/SubMenu.tsx

```diff
@@ -529,7 +517,7 @@ const SubMenu = React.forwardRef(
           {content}
           <IconButton
             data-testid="lg-sub-menu-icon-button"
-            darkMode={true}
+            darkMode={!darkMode}
```

Comment:
is IconButton darkmode broken?

Reply:
I don't think so, we have to use the inverse since the menu becomes light in darkMode.

Reply:
oh you mean why we have to pass darkMode?